.github/CONTRIBUTING.md

```diff
@@ -45,8 +45,10 @@ $ stylua .
 $ luacheck --globals vim busted --no-max-line-length -- .
 ```
 
-4. Make the merge request to the `main` branch of `.gitlab.nvim`
+4. Make the merge request to the `develop` branch of `.gitlab.nvim`
 
 Please provide a description of the feature, and links to any relevant issues. 
 
-That's it! I'll try to respond to any incoming merge request in a few days. Once we've reviewed it and it's been merged into main, the pipeline will detect whether we're merging in a patch, minor, or major change, and create a new tag (e.g. 1.0.12) and release.
+That's it! I'll try to respond to any incoming merge request in a few days. Once we've reviewed it, it will be merged into the develop branc, it will be merged into the develop branch. 
```

Comment:
", it will be merged into the develop branc" is duplicated here

Reply:
Fixing, thanks